extensions/sql-database-projects/src/models/project.ts

```diff
@@ -490,7 +490,7 @@ export class Project implements ISqlProject {
 	//#region Post-deployment scripts
 
 	public async addPostDeploymentScript(relativePath: string): Promise<void> {
-		const result = await this.sqlProjService.addPreDeploymentScript(this.projectFilePath, relativePath);
+		const result = await this.sqlProjService.addPostDeploymentScript(this.projectFilePath, relativePath);
```

Comment:
slop, slop, slop ><

Reply:
my bad too for not catching it in PR. It was a big PR so I probably wasn't looking too closely at each line by this point